### DISCUSSION TITLE
Add decriptions to checkbox items just like radio buttons have.

### DISCUSSION THREAD

**Discussion by aed0nis** [heroui-inc/heroui#669] [Feature requests]
I would love to see descriptions added to checkbox items the same way as radio buttons have now. 

![image](https://user-images.githubusercontent.com/10181188/184502633-8f516ec4-e327-4020-bcf2-ed1c9ee0bcdb.png)

I think bootstrap allows this kind of thing OOB.

**Comment by jrgarciadev** [comment]
Hye @aed0nis thanks for your suggestion, take it for granted 👍🏻
